test/Microsoft.AspNetCore.Server.IISIntegration.Tests/IISMiddlewareTests.cs

```diff
@@ -162,12 +162,10 @@ public void PathBaseHiddenFromServer()
                     app.Run(async context => 
                     {
                         var auth = context.RequestServices.GetRequiredService<IAuthenticationSchemeProvider>();
-                        var ntlm = await auth.GetSchemeAsync("NTLM");
-                        Assert.NotNull(ntlm);
-                        Assert.Equal("Microsoft.AspNetCore.Server.IISIntegration.AuthenticationHandler", ntlm.HandlerType.FullName);
-                        var negotiate = await auth.GetSchemeAsync("Negotiate");
-                        Assert.NotNull(negotiate);
-                        Assert.Equal("Microsoft.AspNetCore.Server.IISIntegration.AuthenticationHandler", negotiate.HandlerType.FullName);
+                        var windows = await auth.GetSchemeAsync("Windows");
```

Comment:
File a followup issue to figure out how we can optionally handle granular schemes. This will be more relevant for HttpSys, but the solution will likely apply to both.